bench/plot-speeds.py

```diff
@@ -27,13 +27,18 @@ def get_values(filename):
     for line in f:
         if line.startswith('-->'):
             tmp = line.split('-->')[1]
-            nthreads, size, elsize, sbits, codec, shuffle = [i for i in tmp.split(', ')]
+            parts = tmp.split(', ')
+            nthreads, size, elsize, sbits, codec, shuffle = parts[:6]
+            safe = 'unsafe'
```

Comment:
For consistency, here it should be 'fast', I think.

Reply:
Fixed.